tests/Functional/Schema/SQLServerSchemaManagerTest.php

```diff
@@ -404,4 +405,30 @@ public function testPkOrdering(): void
         self::assertEquals('colB', $columns[0]);
         self::assertEquals('colA', $columns[1]);
     }
+
+    public function testPartialIndexes(): void
+    {
+        if (! $this->schemaManager->getDatabasePlatform()->supportsPartialIndexes()) {
+            self::markTestSkipped('Platform does not support partial indexes.');
```

Comment:
do we still support MSSQL before 2008?

Reply:
Yes, you're right. All supported SQL Server versions support it now. Will remove it.

Reply:
Fixed